src/vunnel/providers/rootio/parser.py

```diff
@@ -0,0 +1,151 @@
+            self.logger.exception(f"Error downloading Root.io data from {self.url}")
+            raise
+
+    def _normalize(self, distro_name: str, distro_data: dict[str, Any]) -> dict[str, dict[str, Any]]:
+        """Transform Root.io data into OS schema format with unaffected package indicators"""
+        vuln_dict = {}
+
+        distro_version = distro_data.get("distroversion", "unknown")
+        namespace = f"rootio:distro:{distro_name}:{distro_version}"
+
+        for package_data in distro_data.get("packages", []):
+            pkg_info = package_data.get("pkg", {})
+            package_name = pkg_info.get("name", "")
+
+            for cve_id, cve_info in pkg_info.get("cves", {}).items():
+                if cve_id not in vuln_dict:
+                    record = copy.deepcopy(vulnerability.vulnerability_element)
+                    record["Vulnerability"]["Name"] = cve_id
+                    record["Vulnerability"]["NamespaceName"] = namespace
+
+                    # Build reference links
+                    reference_links = vulnerability.build_reference_links(cve_id)
+                    record["Vulnerability"]["Link"] = reference_links[0] if reference_links else ""
+
+                    record["Vulnerability"]["Severity"] = "Unknown"
+                    record["Vulnerability"]["Description"] = f"Vulnerability {cve_id} in {package_name}"
+                    record["Vulnerability"]["FixedIn"] = []
+                    record["Vulnerability"]["Metadata"] = {
+                        "CVE": [{"Name": cve_id, "Link": reference_links[0] if reference_links else ""}],
+                    }
+                    vuln_dict[cve_id] = record
+
+                # Add fixed version info
+                cve_record = vuln_dict[cve_id]
+                fixed_versions = cve_info.get("fixed_versions", [])
+
+                # Determine version format based on distro
+                version_format = self._get_version_format(distro_name)
+
+                # Add the fixed versions
+                # Note: Root.io uses different suffixes for different distros:
+                # - Debian/Ubuntu: .root.io suffix (e.g., 1.5.2-6+deb12u1.root.io.4)
+                # - Alpine: -rXX007X suffix (e.g., -r00071, -r10074)
+                for fixed_version in fixed_versions:
+                    cve_record["Vulnerability"]["FixedIn"].append({
+                        "Name": package_name,
+                        "Version": fixed_version,
+                        "VersionFormat": version_format,
+                        "NamespaceName": namespace,
+                        "VendorAdvisory": {"NoAdvisory": True},
+                    })
+
+                # If no fixed versions, add unfixed entry
+                if not fixed_versions:
+                    cve_record["Vulnerability"]["FixedIn"].append({
+                        "Name": package_name,
+                        "Version": "",  # Empty version indicates no fix available
```

Comment:
I thought we had discussed at https://github.com/anchore/vunnel/pull/863#issuecomment-3461600539 that this vulnerability feed only supports records of fixes? Should this code ever be reachable?